utils/createSiteMeta.ts

```diff
@@ -0,0 +1,60 @@
+const type = 'website'
+const url = 'https://nft.kodadot.xyz'
+const title = 'KodaDot - Kusama NFT Market Explorer'
+const description = 'Creating Carbonless NFTs on Kusama'
+const image = 'https://nft.kodadot.xyz/kodadot_card_root.png'
+
+export default (meta) => {
```

Comment:
I would make the function with a name.

also we can make return type like `SeoMeta[]`

```ts
type SeoMeta = {
  hid: string,
  name: string,
  constent: string
}
```

Reply:
i believe `MetaInfo['meta']` to be the correct return type.